firewood/src/merkle.rs

```diff
@@ -1048,15 +1082,56 @@ mod tests {
         assert!(merkle.root_address().is_none());
     }
 
-    //     #[test]
-    //     fn get_empty_proof() {
-    //         let merkle = create_in_memory_merkle();
-    //         let root_addr = merkle.init_sentinel().unwrap();
+    #[test]
+    fn get_empty_proof() {
+        let merkle = create_in_memory_merkle();
+        let proof = merkle.prove(b"any-key");
+        assert!(matches!(proof.unwrap_err(), MerkleError::Empty));
+    }
+
+    #[test]
+    fn single_key_proof() {
+        let mut merkle = create_in_memory_merkle();
 
-    //         let proof = merkle.prove(b"any-key", root_addr).unwrap();
+        let seed = SystemTime::now()
```

Comment:
Definitely print this seed! If the test fails, the person debugging the test can set the seed manually. Even better, something like this:
```
         let seed = std::env::var("FIREWOOD_TEST_SEED")
             .ok()
             .map_or_else(
                 || None,
                 |s| Some(str::parse(&s).expect("couldn't parse FIREWOOD_TEST_SEED; must be a u64")),
             )
             .unwrap_or_else(|| thread_rng().gen());
```

Reply:
the seed is printed in the helper function that generates the key-value pairs. Changed it to check for the seed env var.